### PR TITLE
global-bindgen-feature

### PR DESCRIPTION
**Pull request by Vixea**

### CODE REVIEW

**Comment by Vixea** [comment]
I love trying to please the ci. Idk on what to do with this last one

File: anvil/Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ profiling = { version = "1.0" }
 
 [dependencies.smithay]
 default-features = false
-features = ["desktop", "wayland_frontend"]
+features = ["desktop", "wayland_frontend", "use_bindgen"]
```
Comment (Drakulix) [inline anvil/Cargo.toml:26]: I don't have anything against this feature, but can we please not enable it for anvil by default. None of our default platforms need it.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -414,7 +414,7 @@ jobs:
           toolchain: nightly-2023-06-17
 
       - name: System dependencies
-        run: sudo apt-get update; sudo apt-get install -y libudev-dev libgbm-dev libxkbcommon-dev libegl1-mesa-dev libwayland-dev libinput-dev libdbus-1-dev libsystemd-dev libseat-dev
+        run: sudo apt-get update; sudo apt-get install -y libdrm-dev libudev-dev libgbm-dev libxkbcommon-dev libegl1-mesa-dev libwayland-dev libinput-dev libdbus-1-dev libsystemd-dev libseat-dev
```
Comment (Drakulix) [inline .github/workflows/ci.yml:417]: Do we need `libdrm-dev`, if we don't enable the feature by default?

Comment (Vixea) [reply .github/workflows/ci.yml:417]: Well let's find out I don't have a way to test the ci locally so it's experiment away

**Review by Drakulix — APPROVED** [review]

**Pull request merged**